apps/portal/components/containers/dashboardLayout/elements/SidebarMenuItems/Favorite.tsx

```
@@ -0,0 +1,114 @@
+import Link from 'next/link';
+import AppSvg from 'components/elements/AppSvg';
+
+export function Chevron(props: any) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="1em"
+      height="1em"
+      viewBox="0 0 1024 1024"
+      {...props}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)